### PR TITLE
Grunt uglify update

### PR DESCRIPTION
**Pull request by jChris85**
- created Gruntfile with grunt uglify task
- updated package.json with grunt devDependencies
- updated link to minified JS file in example page

**Instructions**
- install all npm packages (npm install)
- execute "grunt uglify"

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+            }
+        }
+    });
+
+    // Load grunt uglify taks
+    grunt.loadNpmTasks('grunt-contrib-uglify');
+
+    // Minify task - grunt uglify
+    grunt.registerTask('minify', ['uglify']);
+};
```
Comment (laszlof) [inline Gruntfile.js]: Missing newline at end of file

**Review by laszlof — COMMENTED** [review]
Please use 2 spaces for tabs in the Gruntfile.

**Comment by jChris85** [comment]
Changed the indent from 4 to 2 space. Also added new line at the end of file. However it´s not visible on Github

**Comment by laszlof** [comment]
Thanks. The newline is only visible when its not there. Shows a red circle with an X through it.

**Pull request merged**